Source/Fuse.Navigation/RoutePageProxy.uno

```diff
@@ -65,21 +65,23 @@ namespace Fuse.Navigation
 		}
 		List<Level> _levels = new List<Level>();
 		
+		/** `Init()` must be called to finish initialization. */
 		public RoutePageProxy( Visual source, ProgressUpdated progressUpdated )
 		{
 			_progressUpdated = progressUpdated;
-			Init(source);
+			_source = source;
 		}
 		
+		/** `Init()` must be called to finish initialization. */
 		public RoutePageProxy( Visual source, ActiveChanged activeChanged )
 		{
 			_activeChanged = activeChanged;
-			Init(source);
+			_source = source;
 		}
-		
-		void Init( Visual source )
+
+		/** Call after configuration is completed. This could start the actual registration and messaging */
+		public void Init()
```

Comment:
I can mark these interfaces as internal. I didn't since the old one was technically open, and it's kind of mean to make a new better one that is private and deprecate the old one without a replacement.  Though I'd much prefer for them both to be internal. It's a mechanims that will change.